### PR TITLE
Add verification script, update bindata generation

### PR DESCRIPTION
**Pull request by csrwng**
Updates bindata generation script to not include date metadata (so comparisons work).
Adds verify-bindata script, and travis-ci config

### CODE REVIEW

File: hack/verify-bindata.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+set -euo pipefail
```
Comment (joelddiaz) [inline hack/verify-bindata.sh:2]: no #!/usr/bin/env bash ?

File: hack/verify-bindata.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+echo "Verifying pkg/assets/bindata.go is up to date"
+
+TMP_OUTPUT="$(mktemp)"
+
+REPO_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/.."
```
Comment (joelddiaz) [inline hack/verify-bindata.sh:8]: Took me a while to figure out what this does. You can also `git rev-parse --show-toplevel` to get the root of the git repo.

File: hack/verify-bindata.sh (hunk 1 above)
Comment (csrwng) [reply hack/verify-bindata.sh:2]: Thx fixed

File: hack/verify-bindata.sh (hunk 2 above)
Comment (csrwng) [reply hack/verify-bindata.sh:8]: I like this because it doesn't require a git repo (as in a container build)

**Review by joelddiaz — COMMENTED** [review]
lgtm

**Pull request merged**